src/components/Persona/Persona.tsx

```diff
@@ -23,19 +23,29 @@ export class Persona extends React.Component<IPersonaProps, any> {
   };
 
   public render() {
-    let { className, size, imageUrl, imageInitials, initialsColor, presence, primaryText, secondaryText, tertiaryText, optionalText, hidePersonaDetails } = this.props;
+    let {
+      className,
+      size,
+      imageUrl,
+      imageInitials,
+      initialsColor,
+      presence,
+      primaryText,
+      secondaryText,
+      tertiaryText,
+      optionalText,
+      hidePersonaDetails
+    } = this.props;
 
     return (
-      <div { ... this.props as any } className={ css('ms-Persona', className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence]) }>
+      <div { ...this.props as any } className={ css('ms-Persona', className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence]) }>
```

Comment:
I'm confused. Have you merged latest changes? I thought i had fixed this:

Add this import:

```typescript
import { getNativeProps, divProperties } from '../../Utilities';
```

Then avoid mixing in this.props directly. Instead mix in DIV-safe props only:

```typescript
    let divProps = getNativeProps(this.props, divProperties);

    return <div { ...divProps } ... />
```